display_manager.py

```diff
@@ -42,12 +42,15 @@ class DisplayManager(metaclass=Singleton):
     MSG_WIDTH = SCREEN_WIDTH - BAR_WIDTH - 2
     MSG_HEIGHT = PANEL_HEIGHT - 1
 
+    BACKPACK_WIDTH = 20
+
     game_msgs = []
 
     # TODO: give consoles a better name
     tdl.set_font(get_abs_path('lucida10x10_gs_tc.png'), greyscale=True, altLayout=True)
     console = tdl.Console(MAP_WIDTH, MAP_HEIGHT)
     panel = tdl.Console(SCREEN_WIDTH, PANEL_HEIGHT)
+    backpack = tdl.Console(BACKPACK_WIDTH, SCREEN_HEIGHT)
```

Comment:
Not sure how I feel about having a backpack panel. I mean it's ok for debugging and until we get states, but once we have those wouldn't it be better for it to be rendered only when the player accesses the inventory? (using an inventory panel layout, or maybe just rendering a new panel on top)

Reply:
this is just for now, originally I wanted to have it like in pixel dungeon, this is just to have a backpack that works visually